tests/config.js

```diff
@@ -0,0 +1,37 @@
+module.exports = [{
+	deviceType: `raspberrypi3`,
+	suite: `${__dirname}/../suites/os`,
+	config: {
+		networkWired: false,
+		networkWireless: true,
+		downloadType: 'local',
+		interactiveTests: false, // redundant
+		balenaApiKey: process.env.API_KEY,
+		balenaApiUrl: 'balena-cloud.com',
+		organization: process.env.ORG
```

Comment:
We'll need to an an env variable to the jenkins environment called ORG, with the org name (I think `balena-io`)

Reply:
I'm updating Jenkins and this file, hang tight.

Reply:
Fixup has been pushed and I added an env var for org to Jenkins Leviathan template job properties.
```
BALENA_CLOUD_APP_NAME=testbot-rig
BALENA_CLOUD_ORG=balena-io
```